server/interpreter/data/owner.go

```diff
@@ -18,12 +18,12 @@ package data
 
 // An Owner is an object that can own other objects and properties.
```

Comment:
I am moderately fluent in JavaScript, but I can't figure out what this is from the description. Should I be able to?